source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md

```diff
@@ -17,14 +17,205 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
 
 <div class="toc-list"></div>
 
+## Shopware 5.5
+
+### System requirements changes
+
+The minimum PHP version still is **PHP 5.6.4 or higher**, though **PHP 7.x is highly encouraged**. Please be aware that
+PHP 5.6 and PHP 7.0 reach their "end of life" (the end of the offical support by the PHP Group) by the end of the year.
+For performance and compatibility reasons, we recommend using PHP 7.2 if no Ioncube encrypted plugin are installed.
+
+### Removal of deprecated code
+
+Shopware 5.5 removes a lot of old, deprecated code. If you're updating from Shopware < v5.4.5, you might want to consider
+updating on the latest version of v5.4 first: versions v5.4.5 onwards‚ show deprecation warnings in development mode if a
+deprecated function is being called.
+
+The most relevant changes are:
+
+- The methods `ModelManager::addAttribute` and `ModelManager::removeAttribute` were removed. Use `\Shopware\Bundle\AttributeBundle\Service\CrudService::update` instead.
+- The class `Shopware\Bundle\EmotionBundle\ComponentHandler\EventComponentHandler` was removed. Implement a `\Shopware\Bundle\EmotionBundle\ComponentHandler` instead.
+- The legacy models `Shopware\Models\Customer\Billing` and `Shopware\Models\Customer\Shipping` aren't available anymore.
+The same is true for their references in the `Customer` model (customer.billing, customer.shipping) and the repository
+`Shopware\Models\Customer\BillingRepository`. The matching tables `s_user_billingaddress`, `s_user_shippingaddress`
+and their attribute tables `s_user_billingaddress_attributes` and `s_user_shippingaddress_attributes` won't be removed
+on upgrade until Shopware 5.6, but they won't be kept in sync anymore.
+- The following methods got removed from `Shopware\Models\Payment\Payment`:
+     * `setClass`
```

Comment:
That seems to be wrong :). They are still useable https://github.com/shopware/shopware/blob/5.5/engine/Shopware/Models/Payment/Payment.php#L611

Reply:
Damn, i changed the upgrade.md as well